Changing property raises exception in 1.5.0

I'm testing out 1.5.0 and I ran into the following issue:

I have code that looks like:
```py
f = nix.File.open(filename, nix.FileMode.ReadWrite)

if 'app_logs' not in f.sections:
    f.create_section('app_logs', 'log')
    f.sections['app_logs']['notes'] = ''

if f.sections['app_logs']['notes'] and notes:
    f.sections['app_logs']['notes'] += '\n'
f.sections['app_logs']['notes'] += notes
```

This worked fine in 1.4.9. But with 1.5.0 I get the following error from my pytest tests:

```
>       merger.merge_data(filename, alignment, notes='app notes')

ceed\tests\test_app\test_experiment.py:346:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ ceed\analysis\merge_data.py:1635: in merge_data
    f.sections['app_logs']['notes'] += notes
e:\python\python37\lib\site-packages\nixio\section.py:463: in __setitem__
    prop.values = data
e:\python\python37\lib\site-packages\nixio\property.py:276: in values
    self._h5dataset.shape = np.shape(vals)
e:\python\python37\lib\site-packages\nixio\hdf5\h5dataset.py:109: in shape
    self.dataset.resize(shape)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <HDF5 dataset "notes": shape (1,), type "|O">, size = (1,), axis = None

    def resize(self, size, axis=None):
        """ Resize the dataset, or the specified axis.

        The dataset must be stored in chunked format; it can be resized up to
        the "maximum shape" (keyword maxshape) specified at creation time.
        The rank of the dataset cannot be changed.

        "Size" should be a shape tuple, or if an axis is specified, an integer.

        BEWARE: This functions differently than the NumPy resize() method!
        The data is not "reshuffled" to fit in the new shape; each axis is
        grown or shrunk independently.  The coordinates of existing data are
        fixed.
        """
        with phil:
            if self.chunks is None:
>               raise TypeError("Only chunked datasets can be resized")
E               TypeError: Only chunked datasets can be resized

e:\python\python37\lib\site-packages\h5py\_hl\dataset.py:425: TypeError
```

I imagine this has something to do with https://github.com/G-Node/nixpy/pull/432, but I don't fully understand how/if that disables chunking. Is there perhaps an alternate way to append to a property?

--- a/nixio/cmd/upgrade.py
+++ b/nixio/cmd/upgrade.py
@@ -192,7 +192,7 @@ def update_alias_range_dimension(fname):
 
 
 def create_property(hfile, name, dtype, data, definition=None, unit=None):
-    prop = hfile.create_dataset(name, dtype=dtype, data=data)
+    prop = hfile.create_dataset(name, dtype=dtype, data=data, chunks=True)
     prop.attrs["name"] = name.split("/")[-1]
     prop.attrs["entity_id"] = nix.util.create_id()
     prop.attrs["created_at"] = nix.util.time_to_str(nix.util.now_int())

